src/jaguar.toit

```diff
@@ -275,24 +288,7 @@ run-code image-size/int reader/reader.Reader defines/Map --crc32/int -> none:
   // Write the image into flash.
   image := flash-image image-size reader null defines --crc32=crc32
 
-  // We start the container from a separate task to allow the HTTP server
-  // to continue operating. This also means that the container running
-  // isn't covered by the flashing mutex or associated timeout.
-  task::
-    // First, we wait until we're ready to run the container. Usually,
-    // we are ready right away, but if we've been asked to disable
-    // Jaguar while running the container, we wait until the HTTP server
-    // has been shut down and the network to be free.
-    if disabled: network-free.down
-
-    // Start the image and wait for it to complete.
-    start ::= Time.monotonic-us
-    container ::= run-image image "started" null defines
-    run-to-completion null container start timeout
-
-    // If Jaguar was disabled while running the container, now is the
-    // time to restart the HTTP server.
-    if disabled: container-done.up
+  run-image image "started" null defines
```

Comment:
Maybe add a comment that explains that this returns right away - and doesn't wait for the code to run to completion?